packages/account/src/providers/transaction-request/unknown-transaction-request.ts

```diff
@@ -0,0 +1,57 @@
+import type { BytesLike } from '@fuel-ts/interfaces';
+import type { TransactionUnknown } from '@fuel-ts/transactions';
+
+import { hashTransaction } from './hash-transaction';
+import type { BaseTransactionRequestLike } from './transaction-request';
+import { BaseTransactionRequest, TransactionType } from './transaction-request';
+
+/**
+ * @hidden
+ */
+export interface UnknownTransactionRequestLike extends BaseTransactionRequestLike {
+  data?: BytesLike;
+}
+
+export class UnknownTransactionRequest extends BaseTransactionRequest {
```

Comment:
Why would someone want to create an instance of an unknown transaction?

Reply:
@maschad, I might be missing some context, but I also believe we don't need an Unknown transaction request. If there's something I'm overlooking, please let me know.

Reply:
I second others' thoughts. Ideally, this PR would be [minimal](https://github.com/FuelLabs/fuels-rs/issues/1477#issuecomment-2285907223).
 1. Try/catch unsupported TXs
 1. Ignoring them
 1. Print warning
 1. Move on

Reply:
The `UnknownTransactionRequest` is needed because ultimately because the `sendTransaction` API must return a `TransactionResponse` otherwise it would be an error, in order to construct a `TransactionResponse` you must have a `TransactionRequest` . I've added the `hidden` tag as this is just purely internal mechanics. 

See 6b88425ac

Reply:
As we had discussed the goal is not for consumers to create this type (as they would have created a type that is unknown to the SDK i.e. has a encoded hash >= 6)